<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="kadamba-desktop" name="OpenWorld_Test" tests="3" failures="1" timestamp="27 Jul 2018 09:12:53 GMT" time="13.48" errors="0">
  <testcase name="Test1" time="10.046" classname="TestReport.Login_Test">
    <failure type="java.lang.AssertionError" message="Login page not opened expected [1] but found [0]">
      <![CDATA[java.lang.AssertionError: Login page not opened expected [1] but found [0]
	at TestReport.Login_Test.Test1(Login_Test.java:124)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- Test1 -->
  <testcase name="Test2" time="0.001" classname="TestReport.Login_Test">
    <skipped/>
  </testcase> <!-- Test2 -->
  <testcase name="Test3" time="0.002" classname="TestReport.Login_Test">
    <skipped/>
  </testcase> <!-- Test3 -->
</testsuite> <!-- OpenWorld_Test -->
